app/test.jsx

```diff
@@ -115,26 +125,37 @@ class App extends React.Component {
             high,
             low,
         }] : [];
+
+        const MarkerTimes = [];
+
+        for (let i = 0; i < 5; i++) {
+            MarkerTimes.push(moment().utc().second(0).subtract(i + 3, 'minutes')
```

Comment:
I add sample marker here, as this file is a test file I don't think that makes any problem